tests/conftest.py

```diff
@@ -2924,6 +2924,79 @@ def fixture_demultiplexed_flow_cells_tmp_directory(tmp_path) -> Path:
     return Path(shutil.copytree(original_dir, tmp_dir))
 
 
+@pytest.fixture
+def novaseqx_latest_analysis_version() -> str:
+    return "2"
+
+
+@pytest.fixture
+def novaseqx_flow_cell_dir_name() -> str:
+    return "20230427_LH00188_0001_B223YYCLT3"
+
+
+@pytest.fixture
+def novaseqx_flow_cell_directory(tmp_path: Path, novaseqx_flow_cell_dir_name: str) -> Path:
+    return Path(tmp_path, novaseqx_flow_cell_dir_name)
+
+
+@pytest.fixture
+def demultiplexed_flow_cell_run_directory(tmp_path: Path) -> Path:
+    demultiplexed_runs = Path(tmp_path, "demultiplexed_runs")
```

Comment:
Use a constant to facilitate future refactoring

Reply:
fixed